kong/plugins/opentelemetry/handler.lua

```diff
@@ -121,7 +121,7 @@ function OpenTelemetryHandler:access()
   -- make propagation running with tracing instrumetation not enabled
   if not root_span then
     local tracer = kong.tracing.new("otel")
-    root_span = tracer.start_span("root")
+    root_span = tracer.start_span("kong.request")
```

Comment:
let's rename to `kong` as discussed above
```suggestion
    root_span = tracer.start_span("kong")
```